webpack.config.dev.js

```diff
@@ -0,0 +1,61 @@
+const path = require("path");
+const HtmlWebpackPlugin = require("html-webpack-plugin");
+const { CleanWebpackPlugin } = require('clean-webpack-plugin');
+
+module.exports = {
+  mode: "development",
+  entry: "./src/index.tsx",
+  output: {
+    path: path.join(__dirname, "dev"),
+    filename: '[name].bundle.js',
+    chunkFilename: '[name].bundle.js',
+  },
+  resolve: {
+    modules: [path.resolve(__dirname, './src'), 'node_modules'],
+    extensions: [".tsx", ".ts", ".js"],
+  },
+  devtool: 'source-map',
+  devServer: {
+    static: path.join(__dirname, "src"),
+    compress: true,
+    port: 9000,
+    open: true,
+  },
+  optimization: {
+    splitChunks: {
+      chunks: 'all',
+    },
+  },
+  module: {
+    rules: [
+      {
+        test: /\.(js|jsx)$/,
+        exclude: /node_modules/,
+        use: ["babel-loader"],
+      },
+      {
+        test: /\.(ts|tsx)$/,
+        exclude: /node_modules/,
+        use: ["ts-loader"],
+      },
+      {
+        test: /\.(css|scss)$/,
+        use: [
+          "style-loader",
+          "css-loader",
+          "sass-loader"
+        ],
+      },
+      {
+        test: /\.(jpg|jpeg|png|gif|mp3|svg)$/,
+        use: ["file-loader"],
```

Comment:
DEPRECATED for v5
https://v4.webpack.js.org/loaders/file-loader/

Reply:
Thanks for the tip, it is fixed.